appengine/static/index.html

```diff
@@ -131,6 +131,7 @@
     // Support Libraries
     compile 'com.android.support:appcompat-v7:<span class="frameworkversion">+</span>'
     compile 'com.android.support:cardview-v7:<span class="frameworkversion">+</span>'
+	compile 'com.android.support:design:<span class="frameworkversion">+</span>'
```

Comment:
This change was made with atom and it's annoying to me that atom isn't—by default—smart enough to recognize context and use spaces for tabbing. Also, just the fact that it assumed I wanted a tab character here in general. Psh. I should fix this.